api/src/pcapi/routes/e2e/routes/sandbox_data.py

```diff
@@ -33,6 +33,30 @@ def setup_create_event_individual_offer() -> None:
     offerers_factories.UserOffererFactory(user=pro_user, offerer=venue.managingOfferer)
 
 
+@app.route("/e2e/pro/create-thing-individual-offer", methods=["GET"])
+@spectree_serialize(on_success_status=204)
+def setup_create_thing_individual_offer() -> None:
+    clean_all_database()
+    pro_user = users_factories.ProFactory(
+        lastName="PC Test Pro",
+        firstName="97 0",
+        departementCode="97",
+        postalCode="97100",
+        email="pro_adage_eligible@example.com",
+    )
+    siren = str(MOCK_ADAGE_ELIGIBLE_SIREN)
+    venue = offerers_factories.VenueFactory(
+        managingOfferer__siren=siren,
+        bookingEmail="fake@email.com",
+        comment="Salle de cinéma",
+        siret=f"{MOCK_ADAGE_ELIGIBLE_SIREN}11111",
+        venueTypeCode=VenueTypeCode.MOVIE,
+    )
+    # create bank account to avoid first offer pop up
+    finance_factories.BankAccountFactory(offerer=venue.managingOfferer)
+    offerers_factories.UserOffererFactory(user=pro_user, offerer=venue.managingOfferer)
+
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏Erreur de copier-coller ?
Comme le commit d'avant, on annonce une offre mais on crée un pro et son lieu

Reply:
parreil, setup d'un test E2E